cmd/nv2/gencert.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	KEYS_BASE_DIR = ".notary"
+)
+
+var generateCertCommand = &cli.Command{
+	Name:      "generate-certificates",
+	Usage:     "Generates a test crt and key file.",
+	ArgsUsage: "[host]",
+	Flags: []cli.Flag{
+		&cli.IntFlag{
+			Name:     "rsaBits",
+			Usage:    "rsaBits 2048",
+			Required: false,
+			Value:    2048,
```

Comment:
Might be a good idea to default to at least 3072 to try to buy some futureproofing.

Reply:
Also user-configurable with a sane set of defaults?